sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/README.md

```diff
@@ -0,0 +1,131 @@
+# Azure Cloud Native Cloud Event support for Event Grid
+
+This library can be used to enable publishing the Cloud Native Computing Foundation(CNCF) [CloudEvents][CNCFCloudEvents]
+using the Azure Event Grid library. 
+
+## Getting started
+
+### Prerequisites
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names
+and `<location>` with a valid Azure service location.
+
+```bash
+az eventgrid domain create --location <location> --resource-group <your-resource-group-name> --name <your-resource-name>
```

Comment:
Prerequisites should have information about JDK, Azure subscription, base Event Grid library and then include details about creating the resources using Azure CLI.